### ISSUE TITLE
slight documentation error

### ISSUE THREAD

**Issue by petaflot** [koalaman/shellcheck#2702]
`https://www.shellcheck.net/` says:

```
You can cabal, apt, dnf, pkg or brew install it locally right now. 
```

this should be

```
You can cabal, apt, dnf, pkg, brew or emerge install it locally right now. 
```

**Comment by TinCanTech** [comment]
@petaflot `emerge` is the Gentoo package manager.

**Comment by petaflot** [comment]
@TinCanTech I am fully aware of that.. but how is it relevant?

**Comment by TinCanTech** [comment]
Ha, for some reason i read this as `remove: emerge` not `add: emerge`. Sorry about that!

**Comment by petaflot** [comment]
crazy how we all can sometimes read or hear the exact opposite of a message. no problem, and I'm glad you're not one of those who will start being aggressive when they realize they made a silly mistake.

ps: cool nick :-)

**Comment by petaflot** [comment]
also and while I'm at it, on Gentoo `dev-util/shellcheck` has a massive load of masked dependencies, but there is also a `dev-util/shellcheck-bin` package :-] I suppose it may be worth to mention.

**Comment by port19x** [comment]
I think it's pretty silly to exhaustively list all available native packages for the most starred haskell project on github that exists in every repo under the sun.
It would be a lot more sensible to just remove that line on the website and use a repology badge for the readme instead of listing each and every package manager under the sun.

<details>
 <summary> just look at this madness </summary>
 
![image](https://user-images.githubusercontent.com/82055622/221859129-35ea2e5b-8083-420c-b122-18e1dec8c969.png)
</details

**Comment by petaflot** [comment]
yeah. better write sth like "most distros have it packaged" and add a link to "this madness"
